Mathlib/Geometry/RingedSpace/PresheafedSpace.lean

```diff
@@ -52,22 +53,12 @@ variable {C}
 
 namespace PresheafedSpace
 
--- Porting note: using `Coe` here triggers an error, `CoeOut` seems an acceptable alternative
 instance coeCarrier : CoeOut (PresheafedSpace C) TopCat where coe X := X.carrier
 
 attribute [coe] PresheafedSpace.carrier
 
--- Porting note: we add this instance, as Lean does not reliably use the `CoeOut` instance above
--- in downstream files.
 instance : CoeSort (PresheafedSpace C) Type* where coe X := X.carrier
 
--- Porting note: the following lemma is removed because it is a syntactic tauto
-/-@[simp]
-theorem as_coe (X : PresheafedSpace.{w, v, u} C) : X.carrier = (X : TopCat.{w}) :=
-  rfl-/
-
--- Porting note: removed @[simp] as the `simpVarHead` linter complains
--- @[simp]
 theorem mk_coe (carrier) (presheaf) :
```

Comment:
Not for here but the porting suggests that this should be yanked regardless.

Reply:
You mean something like this?
```suggestion
-- Porting note: removed @[simp] as the `simpVarHead` linter complains,
-- this theorem can probably be removed outright.
theorem mk_coe (carrier) (presheaf) :
```

Reply:
The declaration itself. But out of scope

Reply:
Ok! Then I'll go ahead and merge. Thanks for the review!

Reply:
#21382